src/Slider/Slider.tsx

```diff
@@ -165,7 +181,7 @@ const Slider: React.FunctionComponent<SliderProps> = (props: SliderProps): React
     return (
         <div className={"form-group custom-slider" + (props.className ? ` ${props.className}` : "") + (props.disabled ? " disabled" : "")}>
             {props.label && <label className="custom-label">{props.label}</label>}
-            <div className={"input-field" + (props.labels && props.labels.length ? " has-labels" : "")}>
+            <div className={"input-field" + (props.labels && props.labels.length ? " has-labels" : "") + (props.appearance ? ` ${props.appearance}` : ` ${defaultAppearance}`)}>
```

Comment:
classname dependecy please :D

Reply:
this is angular though so we can just se ngClass lol

Reply:
I think you're confused, this is React🤣 .. you can leave it for now since the `classnames` dependency is happening on the refactoring branch. I will do it there when I get to it.

Reply:
> this is angular though so we can just se ngClass lol

omg i don't know where i am anymore 🤣